chalicelib/services/mongo.py

```diff
@@ -0,0 +1,47 @@
+from pymongo.mongo_client import MongoClient
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+mongo_admin_user = os.getenv('MONGO_ADMIN_USER')
+mongo_admin_password = os.getenv('MONGO_ADMIN_PASSWORD')
```

Comment:
So I attempted a test by manually inputting env variables on our chalice lambda application, but whenever it deploys it, it seems to reset and deletes it to the default `config.json` variables.

There seems to be a AWS service called Systems Manager Parameter Store (https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html) and I think I can try give this an attempt on this branch so we can directly call the env vars on the cloud instead!

Reply:
Sounds good. I can't tell if _you_ have to do this, because I can create a parameter; but I'm not sure if I would be able to deploy it on our app. This does look like the expected solution, though: I think we would just replace our .env with the parameter store and be good to go.